src/data/content/risorse-per-progettare/comprendere/analisi-del-contesto/schematizza-il-contesto-d-uso.yaml

```
@@ -224,32 +224,40 @@ components:
         headingLevel: 3
         listItems:
           - label: Mappa degli attori #C
-            srBefore: "Presentazione google: " #C OPEN FILE FORMAT FOR SCREEN READER ARIA LABEL, eg. PDF, ODX, ...
+            srBefore: "Scarica ODP: " #C OPEN FILE FORMAT FOR SCREEN READER ARIA LABEL, eg. PDF, ODX, ...
```

```suggestion
            srBefore: "Scarica presentazione: " #C OPEN FILE FORMAT FOR SCREEN READER ARIA LABEL, eg. PDF, ODX, ...
```

Forse è meglio così? Specificare il formato rischia di confondere

---

In realtà va specificato il formato. In un mondo ideale anche il peso del file che scarichi. 
Opportunità metterlo in srAfter come " (ODP, 300kb)" per esempio.

---

Oppure metterlo proprio esplicito nel nel testo e non solo per SR. Cmq testeremo questa parte nel prossimi giro di test di usabilità e da lì affineremo.

---

Può anche esserci la via di automatizzare formato e dimensione al build

---

Direi di sì, ma lo lascerei a una feature futura. Non in urgenza.

---

Mi ricordano che la nota va davanti al titolo, e col formato, perché sennò la differenza tra i file collegati nelle icone laddove presente più di un formato (tutte le presentazioni ODP per es., dove sono disponibili anche PPTX e PDF) si capirebbe solo alla fine della frase per uno SR per es.  
 
Quindi per ora direi che va bene così. Poi faremo uno sprint evolutivo as soon as possible cercando di uniformare l'esperienza SR e non SR e semplificare l'approccio con un componente e automatismi ad hoc se sei d'accordo @bfabio .